src/main/resources/data/minecraft/tags/items/piglin_loved.json

```diff
@@ -0,0 +1,9 @@
+{
+  "replace": false,
+  "values": [
+    "sandwichable:golden_kitchen_knife",
+    "sandwichable:chopped_golden_carrot",
+	  "sandwichable:golden_apple_slices",
+	  "sandwichable:enchanted_golden_apple_slices"
```

Comment:
Could the indent be fixed on these lines? That would be great.

Reply:
Oh, that's wacky. When I initially wrote up the thing, it showed those two lines as being not as far over as the first two, so I fixed that, but it must've been behaving strangely. It's fixed now- or, at least, it should be.~